tests/app/core/models/session-model.spec.ts

```diff
@@ -52,7 +40,7 @@ describe('Session Model', () => {
     });
 
     it('should throw error on session data with invalid open phases', () => {
-      of({ ...validSessionData, openPhases: ['unknownPhase'] })
+      of({ ...MODERATION_PHASE_SESSION_DATA, openPhases: ['unknownPhase'] })
```

Comment:
Definitely not an issue but just feels weird using the valid session data for Moderation Phase as the default haha 😅 given that its the last phase in the application. But yeah no issues with it, works fine 👍

Reply:
Ahhh, should we create the default as `BUG_REPORTING_PHASE`_SESSION_DATA`? I think then it would be useful to adapt to various other variations from there!

Reply:
Hmm yeah I guess that works, either start with bug-reporting or start with empty? But its not a big problem using Moderation here also haha.

Reply:
Alright! I think we can start with bug reporting phase!